apps/testapp/src/components/RpcMethods/index.ts

```diff
@@ -18,9 +18,11 @@ const ethAccounts = {
 const switchEthereumChain = {
   method: 'wallet_switchEthereumChain',
   params: [{ key: 'chainId', required: true }],
-  format: (data: Record<string, string>) => ({
-    chainId: `0x${Number(data.chainId).toString(16)}`,
-  }),
+  format: (data: Record<string, string>) => [
```

Comment:
Did the switch chain format change?

Reply:
@cb-jake i had to make this as an array to make it work.
will update other cards on the following pr.
![image](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/3dc35236-57db-4b12-9f2e-3bed65c77942)
https://eips.ethereum.org/EIPS/eip-3326#examples

Reply:
Weird that it was working :shrug: Sounds good.

Reply:
updated all other methods as well.